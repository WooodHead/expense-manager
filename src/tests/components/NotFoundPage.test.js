// react-test-renderer
import React from 'react';
import { shallow } from 'enzyme';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import NotFoundPage from '../../components/NotFoundPage';
import toJson from 'enzyme-to-json';

test('should render header properly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});
