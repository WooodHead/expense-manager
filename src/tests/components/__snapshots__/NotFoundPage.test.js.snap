// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header properly 1`] = `
<div>
  404! - 
  <Link
    replace={false}
    to="/"
  >
     Go Home
  </Link>
</div>
`;
